kukaPoint[] track
int32 speed # mm/s
int32 mod # movel 1  movej 2
---
bool ok
